.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] `.gitignore` should contain a `test.md` file, so it can't be added by accident to your repo